src/components/Link/Link.tsx

```diff
@@ -0,0 +1,32 @@
+import clsx from 'clsx';
+import { Link as ReactLink} from 'react-router-dom';
+
+export type LinkProps = {
+  children: React.ReactNode;
+  type?: 'primary' | 'secondary' | 'default';
+  className?: string;
+  to?:string;
+} 
+
+export default function Link({ 
+  className, 
+  children, 
+  to = '', 
+  type = 'default',
+  ...rest 
+}: LinkProps) {
```

Comment:
please align this component to others.
```suggestion
export default function Link(props: LinkProps) {
const { 
  className, 
  children, 
  to = '', 
  type = 'default',
  ...rest 
} = props;
```